dotcom-rendering/src/web/components/TopicFilterBank.tsx

```diff
@@ -116,8 +116,8 @@ export const TopicFilterBank = ({
 				Filters <span css={headlineAccentStyles(palette)}>BETA</span>
 			</div>
 			<div css={topicStyles}>
-				{hasKeyEvents ? (
-					<FilterButton
+				{keyEvents && keyEvents.length > 0 ? (
```

Comment:
I think we can keep `hasKeyEvents`, here, which is a strict boolean.

```suggestion
				{hasKeyEvents ? (
```